docs/reference/troubleshooting/common-issues/task-queue-backlog.asciidoc

```diff
@@ -1,103 +1,117 @@
 [[task-queue-backlog]]
 === Task queue backlog
 
-A backlogged task queue can prevent tasks from completing and put the cluster
-into an unhealthy state. Resource constraints, a large number of tasks being
-triggered at once, and long running tasks can all contribute to a backlogged
-task queue.
+*******************************
+*Product:* Elasticsearch +
+*Deployment type:* Elastic Cloud (hosted or self-managed), self-managed +
+*Versions:* All
+*******************************
+
+A backlogged task queue can prevent tasks from completing and lead to an 
+unhealthy cluster state. Contributing factors include resource constraints, 
+a large number of tasks triggered at once, and long-running tasks.
 
 [discrete]
 [[diagnose-task-queue-backlog]]
-==== Diagnose a task queue backlog
+==== Diagnose a backlogged task queue
 
 **Check the thread pool status**
 
 A <<high-cpu-usage,depleted thread pool>> can result in
 <<rejected-requests,rejected requests>>. 
 
-Thread pool depletion might be restricted to a specific <<data-tiers,data tier>>. If <<hotspotting,hot spotting>> is occuring, one node might experience depletion faster than other nodes, leading to performance issues and a growing task backlog.
-
-You can use the <<cat-thread-pool,cat thread pool API>> to see the number of
-active threads in each thread pool and how many tasks are queued, how many
-have been rejected, and how many have completed.
+Use the <<cat-thread-pool,cat thread pool API> to monitor
+active threads, queued tasks, rejections, and completed tasks:
 
 [source,console]
 ----
 GET /_cat/thread_pool?v&s=t,n&h=type,name,node_name,active,queue,rejected,completed
 ----
 
-The `active` and `queue` statistics are instantaneous while the `rejected` and
-`completed` statistics are cumulative from node startup.
+* Look for high `active` and `queue` metrics, which indicate potential bottlenecks.
+* Analyze whether thread pool issues are specific to a <<data-tiers,data tier>> or
+caused by uneven node resource utilization such as <<hotspotting,hot spotting>>.
```

Comment:
uneven node resource utilization is synonymous with hot spotting. would rephrase this to not be "caused by uneven node resource utilization" but instead as something to look for. hack edit:

```suggestion
* Analyze whether thread pool issues are specific to a <<data-tiers,data tier>>.
* Analyze whether a specific node's thread pool is depleting faster than others. This can be caused by uneven node resource utilization, also known as <<hotspotting,hot spotting>>.
```

stray thoughts: 
- wonder if we should even introduce hot spotting here or leave it to the resolution
- wonder if "analyze" is too loaded of a word (overthinking here). people might think it's harder than it is, when it's really a "check if"
- little red flag in the back of my mind here - in some cases, we can already tie diagnostics to resolutions at this stage. should we be tying them together somehow? 
- is hotspotting missing from the recommendations?